src/lightning_lite/accelerators/cuda.py

```diff
@@ -78,10 +78,61 @@ def register_accelerators(cls, accelerator_registry: Dict) -> None:
         )
 
 
-def _get_all_available_cuda_gpus() -> List[int]:
+def find_usable_cuda_gpus(num_gpus: int = -1) -> List[int]:
```

Comment:
I suggest using "devices" instead of "gpus" for consistency with the trainer flag

```suggestion
def find_usable_cuda_devices(num_devices: int = -1) -> List[int]:
```

If you agree, this should also be changed in the docstrings etc